plugin.video.retrospect/resources/menus/menu_install_widevine.py

```diff
@@ -0,0 +1,46 @@
+#===============================================================================
+# LICENSE Retrospect-Framework - CC BY-NC-ND
+#===============================================================================
+# This work is licenced under the Creative Commons
+# Attribution-Non-Commercial-No Derivative Works 3.0 Unported License. To view a
+# copy of this licence, visit http://creativecommons.org/licenses/by-nc-nd/3.0/
+# or send a letter to Creative Commons, 171 Second Street, Suite 300,
+# San Francisco, California 94105, USA.
+#===============================================================================
+import sys
+
+import xbmcgui
+import xbmcaddon
+
+
+def install_widevine():
```

Comment:
inputstreamhelper should do this automatically on your behalf

Reply:
That could be, but I personally don't like that behaviour, so users can to this from my context menu. It will then kick-off the inputstreamhelper code manually. As I run their code, it still asks all the questions and confirmations.